weather_mv/loader_pipeline/ee.py

```diff
@@ -364,13 +375,23 @@ def __init__(self,
                          private_key=private_key,
                          service_account=service_account,
                          use_personal_account=use_personal_account)
+        self.asset_location = asset_location
         self.ee_asset = ee_asset
+        self.ee_asset_type = ee_asset_type
+        self.force_overwrite = force
 
     def process(self, uri: str) -> t.Iterator[str]:
         """Yields uri if the asset does not already exist."""
         self.check_setup()
-
         asset_name = get_ee_safe_name(uri)
+
+        # Checks if the asset is already present in the GCS bucket or not.
+        target_path = os.path.join(
+            self.asset_location, f'{asset_name}{ASSET_TYPE_TO_EXTENSION_MAPPING[self.ee_asset_type]}')
+        if not self.force_overwrite and FileSystems.exists(target_path):
+            logger.info(f'Asset file {target_path} already exists in GCS bucket. Skipping...')
```

Comment:
It would appear this is going to just drop the file if the matching COG already exists, regardless of the state of the asset in EE. That's fine, but I think the readme description of what happens could be more clear.

Reply:
Updated the README statement.